src/service/ampdoc-impl.js

```diff
@@ -414,6 +419,25 @@ export class AmpDoc {
     return v == null ? null : v;
   }
 
+  /**
+   * Returns the value of an ampdoc's meta tag content for a given name, or
+   * `null` if the meta tag does not exist. To be implemented by subclasses.
+   * @param {string} unusedName
+   * @return {?string}
+   */
+  getMetaByName(unusedName) {
+    return /** @type {?} */ (devAssert(null, 'not implemented'));
+  }
+
+  /**
+   * Stores the value of an ampdoc's meta tag content for a given name.
```

Comment:
Let's add some language that dissuades users from calling this. It should only be used during setup of the AmpDoc.